lib/gsheets/gsheets.js

```diff
@@ -1,21 +1,25 @@
-var fs = require('fs');
-var readline = require('readline');
-var google = require('googleapis');
-var GoogleAuth = require('google-auth-library');
-var process = require('process');
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
@paulirish what do you think if we implement logic this file as a `Class` ? It will be easier to test + more functionality will be covered...